app/src/theme.ts

```diff
@@ -107,7 +107,7 @@ export const buildNotdTheme = (): ITheme => {
         'border-radius': '0',
       },
       borderColored: {
```

Comment:
this doesnt really make sense - it doesnt even hav a border? can you find where its used pls